README.md

```diff
@@ -19,29 +19,31 @@ Our aim is to design a DL model that is pre-trained on visual imagery data (e.g.
 
 ## Poetry
 
-This project uses [poetry](https://python-poetry.org/) for dependency management. To install poetry, follow the instructions on the [poetry website](https://python-poetry.org/docs/#installation).
+This project uses [conda](https://docs.conda.io/en/latest/) and [pip](https://pip.pypa.io/en/stable/) for package management.
+
+```bash
 
 To create a virtual environment and install the dependencies, run
 
 ```bash
-poetry install --no-root
+$ conda create -n dtu_mlops_project python=3.11
```

Comment:
we could also store the environment using conda directly (in an `env.yaml` file extracted using `conda env export > env.yaml`. This way all these instructions could be simplified into `conda env create -f env.yaml`

Reply:
forgot to mention: here's how to install pytorch + torchaudio/vision using conda (from https://pytorch.org/get-started/locally/): `conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia`

Reply:
I tried installing the branch br/conda with the new install instructions in the readme and that worked (i.e. make train worked). So I'd say this requirements.txt pip-based install is fine. In the past I had some experiences with a conda env.yaml where there were issues with transferring between different os.

Reply:
alright, let's stick to this then :)

Reply:
I also had an issue with conda when installing certain packages, they don't have some packages so I think pip's better